README.md

```diff
@@ -11,7 +11,7 @@ These steps assume that you [have installed git locally](https://www.atlassian.c
 
 2. Clone your forked repo locally. `git clone git@github.com:<your-github-handle>/git-tuto.git`
 
-3. Add the original project as a distant repo. `git remote add https://github.com/zaccharieramzi/git-tuto.git`
+3. Add the original project as a distant repo. `git remote add master https://github.com/zaccharieramzi/git-tuto.git`
```

Comment:
Actually, the preferred name for the original project distant repo is `upstream`. It is also how I called it in step 10., so I'd rather you change it to that.

But thanks for the change!